company/how-to-guides-for-staff/how-to-purchase/how-to-arrange-a-bounty-program.md

```diff
@@ -29,7 +29,7 @@ Once the bounty lead and bounty contributor have agreed to the terms, please hav
 * Sign the **\[Bounty Agreement\]**
```

Comment:
There isn't a link to the Bounty Agreement.

Reply:
this is a placeholder as this will need to come from HR